elasticsearch5/search-index/src/main/java/org/vertexium/elasticsearch5/ElasticsearchSearchQueryBase.java

```diff
@@ -480,9 +481,12 @@ protected IdStrategy getIdStrategy() {
 
     private void closeScroll(String scrollId) {
         try {
-            client.prepareClearScroll()
+            ClearScrollResponse clearScrollResponse = client.prepareClearScroll()
                 .addScrollId(scrollId)
                 .execute().actionGet();
+            if (!clearScrollResponse.isSucceeded()) {
+                LOGGER.warn("Unable to clear scroll " + scrollId);
```

Comment:
any possibility of logging the response here

Reply:
Looking at the `ClearScrollResponse`, it appears to only have two fields: `succeeded` and `num_freed`. I don't think we can provide any other help in that log message